prow/plugins/trigger/push.go

```diff
@@ -17,12 +17,44 @@ limitations under the License.
 package trigger
 
 import (
+	"fmt"
+	"strings"
+
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/plank"
 )
 
+var mergeRe = `Automatic merge from submit-queue`
+
+func notifyOnSlackIfManualMerge(sc SlackClient, pe github.PushEvent) error {
+	if sc == nil {
+		return nil
+	}
+	sendMessage := false
+	for _, commit := range pe.Commits {
+		if strings.Contains(commit.Message, mergeRe) {
```

Comment:
I don't think I understand what you're saying here. I agree you should send a message only if the author is not in the whitelist, where the whitelist is `["k8s-merge-robot"]` for now.